packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.types.ts

```diff
@@ -94,6 +108,57 @@ export interface IScrollablePaneStyles {
   contentContainer: IStyle;
 }
 
+/**
+ * {@docCategory ScrollablePane}
+ */
+export interface IStickyContainerBehavior {
+  /**
+   * If true, it replicates actual element instead of keeping placeholder for the component which is to be sticky'ed.
+   * Calculating placeholder height & width could be an expensive operation.
+   * It's a trade off- cost of replicating the element vs. cost of calculating placeholder height & width.
+   */
+  notUsePlaceHolder: boolean;
+
+  /**
+   * If true, arranges Sticky component(s) based on Sticky's 'order' prop in ascending order.
+   */
+  arrangeStickiesBasedOnOrder: boolean;
+
+  /**
+   * Determies when sticky behavior kicks in for Sticky component(s).
+   * There are some calculations which determine if a Sticky component is sticky or non-sticky.
+   * These calculations can be expensive and affect page load time.
+   */
+  containerBehavior: StickyContainerBehaviorType;
+}
+
+/**
+ * {@docCategory ScrollablePane}
+ */
+export enum StickyContainerBehaviorType {
+  /**
+   * This is the default behavior. It can affect page load time.
```

Comment:
Page load time seems like an app-level concern not a UI library's re: documentation.